From 906cd47ce7dc14007997ee4f368701ec0c41ca80 Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Thu, 17 May 2012 17:33:49 -0700
Subject: [PATCH 272/859] ASoC: sh: fsi: use register field macro name on
 IN/OUT_DMAC

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Signed-off-by: Mark Brown <broonie@opensource.wolfsonmicro.com>
(cherry picked from commit 4d62ec125be98f5a446444e4ce53f4f4e2e1baff)

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 sound/soc/sh/fsi.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/sh/fsi.c b/sound/soc/sh/fsi.c
index d759314..2582663 100644
--- a/sound/soc/sh/fsi.c
+++ b/sound/soc/sh/fsi.c
@@ -1218,8 +1218,8 @@ static int fsi_hw_startup(struct fsi_priv *fsi,
 	 * FSI2 chip can select it.
 	 */
 	if (fsi_ver >= 2) {
-		fsi_reg_write(fsi, OUT_DMAC,	(1 << 4));
-		fsi_reg_write(fsi, IN_DMAC,	(1 << 4));
+		fsi_reg_write(fsi, OUT_DMAC,	VDMD_BACK);
+		fsi_reg_write(fsi, IN_DMAC,	VDMD_BACK);
 	}
 
 	/* irq clear */
-- 
1.8.1.4

